src/render/RenderContext.php

```diff
@@ -32,6 +33,15 @@ public function disableExtensions(): this {
     return $this;
   }
 
+  public function addNoFollowUGCAllLinks(): this {
```

Comment:
Is the setter which assigns false intentionally missing?
This appears to be a pattern with most behavior switches in this library (you can not set the default state).

Reply:
Yeah, I noticed that pattern in the existing code and kept following, but am wondering what FB thinks about adding setters that assign false!

Reply:
With this enum, should I keep the other functions I created public (like changing allowlist URIs, disabling image filtering) in case people want to fiddle with the settings?

Reply:
Yep - the purpose of the enum would be to set reasonable defaults easily; using that as a starting point for customization via the other methods makes sense